scripts/travis_script.sh

```diff
@@ -1,4 +1,6 @@
 #!/bin/bash
+# make sure we have the typings
+ls typings/codemirror || exit 1
```

Comment:
No, that's supposed to be there.  Let's see if it picks up my cache this time...